src/replication.c

```diff
@@ -2837,6 +3337,27 @@ check_for_command_completion(spec_t *spec, rcommon_cmd_t *rcomm_cmd, ISTGT_LU_CM
 
 		if (rcomm_cmd->resp_list[i].status & SENT_TO_HEALTHY)
 			healthy_replica++;
+
+		if (spec->rebuild_info.dw_replica != NULL &&
+		    rcomm_cmd->resp_list[i].replica == spec->rebuild_info.dw_replica) {
+			found_in_list = 1;
+		}
+	}
+	/* In scale up replica cases there is reason to
+	 */
+	if (found_in_list == 0) {
+		ignore_transition_state_resp = can_transition_resp_ignore(spec);
```

Comment:
is there any flag to say that replica is under transition from unknown to known list?

Reply:
currently, we are not maintaining